mediapipe/graphs/instantmotiontracking/subgraphs/calculators/tracked_anchor_manager_calculator.cc

```diff
@@ -0,0 +1,154 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "mediapipe/framework/calculator_framework.h"
+#include "mediapipe/framework/formats/detection.pb.h"
+#include "mediapipe/framework/formats/location_data.pb.h"
+#include "mediapipe/framework/port/status.h"
+#include "mediapipe/framework/port/ret_check.h"
+#include "mediapipe/util/tracking/box_tracker.h"
+#include "mediapipe/graphs/instantmotiontracking/calculators/transformations.h"
+
+namespace mediapipe {
+
+constexpr char kAnchorsTag[] = "ANCHORS";
+constexpr char kBoxesInputTag[] = "BOXES";
+constexpr char kBoxesOutputTag[] = "START_POS";
+constexpr char kCancelTag[] = "CANCEL_ID";
+
+// Generates the final model matrix rotations via the IMU (orientation sensors)
+// of the device and user rotation information
+//
+// Input:
+//  ANCHORS - Initial anchor data (tracks changes and where to re/position)
```

Comment:
Same as other calculators' streams: let's add in the type information, and whether these are required/optional, and if required we can add them to the checks in GetContract()